bundles/pixi.js-node/src/adapter/adapter.ts

```diff
@@ -36,7 +36,7 @@ export const NodeAdapter = {
 
         return new Promise((resolve, reject) =>
         {
-            const filePath = utils.path.normalize(request.url);
+            const filePath = utils.path.normalize(typeof url === 'string' ? url : request.url);
```

Comment:
```suggestion
            const filePath = utils.path.normalize(decodeURI(request.url));
```

Perhaps this is better?

Reply:
Yes, but only if the Request instance doesn't modify the path. (I haven't looked into that)

Reply:
```js
const request = new Request('a%20b c.txt');
console.log(request.url); // a%20b%20c.txt
```

Reply:
Yeah, I think circumventing `request.url` (not using decodeURI) is probably a better idea, since Request can do thing to resolve paths, add host name, etc.